core/src/config.rs

```diff
@@ -139,10 +139,11 @@ mod tests {
 
     use super::*;
 
-    const CONFIGURATION_PATH: &str = "tests/test_config.json";
-    const TRUSTED_PEERS_PATH: &str = "tests/test_trusted_peers.json";
+    const CONFIGURATION_PATH: &str = "../configs/peer/config.json";
+    const TRUSTED_PEERS_PATH: &str = "../configs/peer/trusted_peers.json";
 
     #[test]
+    #[ignore = "We shouldn't be testing `serde`"]
```

Comment:
Should we just remove these tests?

Reply:
I've re-purposed them to validate the configuration used for docker.